tensorflow/core/util/mkl_util.h

```diff
@@ -528,29 +699,52 @@ inline Tensor ConvertMklToTF(OpKernelContext* context, const Tensor& mkl_tensor,
     context->allocate_temp(DataTypeToEnum<T>::v(), output_shape,
                            &output_tensor);
 
+#ifdef ENABLE_MKLDNN_V1
+    engine cpu_engine(engine::kind::cpu, 0);
+    stream cpu_stream(cpu_engine);
+#else
     auto cpu_engine = engine(engine::cpu, 0);
+#endif
     MklDnnData<T> input(&cpu_engine);
 
     // Get Mkl layout of input tensor.
     auto input_mkl_md = mkl_shape.GetMklLayout();
     auto output_tf_md = mkl_shape.GetTfLayout();
+#ifndef ENABLE_MKLDNN_V1
+    // Memory primitive descriptor is deprecated in MKL-DNN v1.0.
     auto output_tf_pd = memory::primitive_desc(output_tf_md, cpu_engine);
+#endif
     input.SetUsrMem(input_mkl_md, &mkl_tensor);
 
-    // reorder
+#ifdef ENABLE_MKLDNN_V1
+    // Reorder
+    if (input.IsReorderNeeded(output_tf_md)) {
+      std::vector<primitive> net;
+      std::vector<MemoryArgsMap> net_args;
+      DCHECK_EQ(input.CheckReorderToOpMem(output_tf_md, &output_tensor, net,
```

Comment:
`DCHECK(expression)` checks if `expression` is true. You can use `DCHECK` instead of `DCHECK_EQ`.

Reply:
Done